src/Pages/SignUp/PasswordRegister/PasswordRegister.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import SignUpNav from '../../../Components/SignUpNav/SignUpNav';
+import InputBox1 from '../../../Components/InputBox/InputBox1';
+import InputBox2 from '../../../Components/InputBox/InputBox2';
+import ResidentNumber from '../../../Components/ResidentNumber/ResidentNumber';
+import styled from 'styled-components';
+import { flexCenter, theme } from '../../../Styles/CommonStyle';
+
+const PasswordRegister = () => {
+  const [password, setPassword] = useState('');
+  const [passwordCheck, setPasswordCheck] = useState('');
+  const [validPw, setValidPw] = useState(true);
+  const [validDoubleCheck, setValidDoubleCheck] = useState(true);
+
+  //비밀번호, 비밀번호 재확인 인증 체크
+  const validationCheck = (name, value) => {
+    if (name === 'passwordBox') {
+      let passwordRules = /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,16}$/;
+      const validPw = passwordRules.test(value);
+      setValidPw(validPw);
+      return;
+    }
+    if (name === 'passwordRecheck') {
+      const validDoubleCheck = password === value;
+      setValidDoubleCheck(validDoubleCheck);
+      return;
+    }
+  };
+
+  const handlePassword = (e) => {
+    const { name, value } = e.target;
+    validationCheck(name, value);
+    setPassword(value);
+  };
+
+  const handlePasswordCheck = (e) => {
+    const { name, value } = e.target;
+    validationCheck(name, value);
+    setPasswordCheck(value);
+  };
+
+  return (
+    <div>
+      <SignUpNav name={'비밀번호 등록'} />
+      <Wrap>
+        <div className="inputContainer">
+          <InputBox1
+            value={password}
+            type={'password'}
+            header={'로그인 시 사용할 비밀번호'}
+            placeholder={'비밀번호 입력'}
+            text={'영문,숫자를 포함한 8~16자 조합으로 입력해 주세요.'}
+            handlePassword={handlePassword}
+            validPw={validPw}
+          />
+          <InputBox2
+            value={passwordCheck}
+            type={'password'}
+            placeholder={'비밀번호 재입력'}
+            text={'비밀번호 확인을 위해 비밀번호를 다시 한 번 입력해 주세요.'}
```

Comment:
```suggestion
            type='password'
            placeholder='비밀번호 재입력'
            text='비밀번호 확인을 위해 비밀번호를 다시 한 번 입력해 주세요.'
```
일반 스트링으로 그냥 넘겨줄 수 있는 내용은 중괄호 제거해주세요. 자바스크립트로 된 내용을 입력할 때만 붙여주시면 됩니다.